### PR TITLE
Feature/reol 263 newsletter signup

### PR DESCRIPTION
**Pull request by mian3010**
Not yet ready to be merged.

### CODE REVIEW

File: sites/all/drush/englishdevel.drush.inc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * Hack to switch language to i18n source language or English in drush.
+ */
+function englishdevel_drush_init() {
+  if (function_exists('language_list')) {
+    global $language;
+    $languages = language_list();
+    $prefix = variable_get('i18n_string_source_language', 'en');
+    $language = $languages[$prefix];
+  }
+}
```
Comment (mian3010) [inline sites/all/drush/englishdevel.drush.inc]: Er det OK at denne bliver comitted med, eller skal jeg lade være med det?

Comment (xendk) [inline sites/all/drush/englishdevel.drush.inc]: I'll suggest you install https://github.com/arnested/drush_features_language instead.

Or upgrade Features to the version that fixes that issue. :-)

**Comment by xendk** [comment]
Looks fine.

File: sites/all/drush/englishdevel.drush.inc (hunk 1 above)
Comment (mian3010) [inline sites/all/drush/englishdevel.drush.inc]: I do not think upgrading features is within the scope of this issue. Would it be okay to commit Arnes, or should i just keep it on my local environment?

**Comment by xendk** [comment]
Still looks fine.

File: sites/all/drush/englishdevel.drush.inc (hunk 1 above)
Comment (xendk) [inline sites/all/drush/englishdevel.drush.inc]: Just keep it locally.

Comment (mian3010) [inline sites/all/drush/englishdevel.drush.inc]: Removed

**Comment by xendk** [comment]
Superceeded by #104 (which is basically a rebase)

**Pull request closed without merging**